plugin.video.themoviedb.helper/resources/lib/script/router.py

```diff
@@ -331,6 +385,27 @@ def sort_list(**kwargs):
     xbmc.executebuiltin(format_folderpath(encode_url(**kwargs)))
 
 
+def recache_image(recache_image, **kwargs):
```

Comment:
Highly doubt this is allowed...

Reply:
That's frustrating. Is there some built-in method I can use to ask the texture cache to recache a local image?

Very occasionally a corrupted version of a PIL generated blur image ends up in the texture cache because Kodi starts loading the file before the filesystem has finished writing. This happens despite having done a `flush()` `os.fsync()` and `close()`  in [_saveimage()](https://github.com/xbmc/repo-plugins/blob/1acc518b219c90dc0bcd19fbf4634175d898161b/plugin.video.themoviedb.helper/resources/lib/monitor/images.py#L89-L98) before passing the uri to a window property.

Happy to remove if there's a better approach to forcing a manual recache if needed. OR, ideally, if there's an improvement I can make to my `_saveimage()` function to avoid texture cache corruption. I'm just stumped as to what since I'm already working on a temp file to do transformations first before saving to the actual output file and I never rewrite an existing file once generated.

Reply:
Either way, I'll remove it for now. I'll force push an update soon.